themes/classic/_dev/css/components/imageslider.scss

```diff
@@ -73,3 +57,52 @@
     }
   }
 }
+
+@include media-breakpoint-down(lg) {
+  .carousel {
+    .carousel-inner {
+      height: auto;
+    }
+  }
+}
+
+@include media-breakpoint-down(md) {
+  .carousel {
+    .carousel-item {
+      .caption {
+        bottom: calc(50% - 2rem);
+        left: 10%;
+      }
+      .caption {
+        .display-1 {
+          font-size: 2rem;
+        }
+      }
+      .caption-description {
+        display: none;
+      }
+    }
+  }
+}
+
+@include media-breakpoint-down(sm) {
+  .carousel {
+    .carousel-item {
+      .caption {
+        bottom: calc(50% - 1.5rem);
+      }
+      .caption {
+        .display-1 {
+          font-size: 1.5rem;
+        }
+      }
+    }
+    .carousel-control {
+      .icon-prev, .icon-next {
+        i {
+          font-size: 2rem;
+        }
+      }
+    }
+  }
+}
```

Comment:
Missing new line at the end of file.

Reply:
@PierreRambaud added in https://github.com/PrestaShop/PrestaShop/pull/10341/commits/15e7d09021d3e8e218b04fbc6cf60496cb82fd4d